### PR TITLE
Docs: add new component chapters

### PR DESCRIPTION
**Pull request by hrach**

### CODE REVIEW

File: docs/03-components/05-input/checkbox.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Android
+---
+
+## Overview
+
+`Checkbox` component is provided in two flavors, as a simple checkbox and as a row with an additional label and a description:
+
+- [`Checkbox`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-checkbox.html) - a simple checkbox
+- [`CheckboxField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-checkbox-field.html) a checkbox with label and description
```
Comment (Drekorian) [inline docs/03-components/05-input/checkbox.md:10]:
```suggestion
- [`CheckboxField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-checkbox-field.html) a checkbox with a label and a description
```

File: docs/03-components/05-input/checkbox.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Usage
+
+`Checkbox` accepts a boolean state and a callback on its change.
```
Comment (Drekorian) [inline docs/03-components/05-input/checkbox.md:14]:
```suggestion
`Checkbox` accepts a `Boolean` state and a callback on its change.
```

File: docs/03-components/05-input/checkbox.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+```kotlin
+@Composable
+fun Example() {
+    Checkbox(
+        checked = false,
+        onCheckedChange = { /*...*/ },
```
Comment (Drekorian) [inline docs/03-components/05-input/checkbox.md:21]: This is just a personal preference. I think `{}` demonstrates the parameter is a lambda sufficiently. The comment doesn't really add too much, what do you think?

```suggestion
        onCheckedChange = {},
```

File: docs/03-components/05-input/checkbox.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        description = { Text("If selected, you will get the latest updates.") }
+    ) {
+        Text("Autoupdate")
+    }
+}
+```
+
+## UI testing
+
+Slotting API allows you to add testTag to particular component parts. Utilize `assertIsOn` and `assertIsOff` to check the state.
```
Comment (Drekorian) [inline docs/03-components/05-input/checkbox.md:43]:
```suggestion
The slotting API allows you to add a testTag to particular component parts. Utilize `assertIsOn` and `assertIsOff` to check the state.
```

File: docs/03-components/05-input/inputfield.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+To customize Password field, you may want to show
+
+- [`PasswordStrengthIndicator`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-password-strength-indicator.html)
+
+`TextField` represents an atomic block of a label, an input and an info/error message.
+
+## Usage
+
+`TextField` accepts a text value and a callback for its change. Read how to [manage text field state](https://medium.com/androiddevelopers/effective-state-management-for-textfield-in-compose-d6e5b070fbe5). 
+Provide a label, optionally you can pass a placeholder, leading & trailing icons, including their callbacks, an error or info message.
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfield.md:22]:
```suggestion
Provide a label. Optionally, you may pass a placeholder, leading & trailing icons with their respective callbacks, or an error or an info message.
```

File: docs/03-components/05-input/inputfield.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+```kotlin
+@Composable
+fun Example() {
+    TextField(
+        value = "...",
+        onValueChange = { /* ... */ },
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfield.md:29]: _LOGAF_ < 1.
```suggestion
        onValueChange = {},
```

File: docs/03-components/05-input/inputfield.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+        error = if (value.length > 3) {
+            @Composable { Text("Error") }
+        } else {
+            null
+        },
+    )
+}
+```
+
+To utilize password strength indicator, use the `strengthContent` slot on `PasswordTextField` composable, evaluate the strength as a `0f <= strength <= 1f` and map this strength to predefined colors, alternatively pass custom colors.
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfield.md:53]:
```suggestion
To utilize the password strength indicator, use the `strengthContent` slot on `PasswordTextField` composable, evaluate the strength as a `0f <= strength <= 1f` and map this strength to predefined colors. Alternatively, pass custom colors.
```

File: docs/03-components/05-input/inputfield.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+    PasswordTextField(
+        value = value,
+        onValueChange = { value = it },
+        strengthContent = {
+            val strength = calcStrength(value)
+            PasswordStrengthIndicator(
+                modifier = Modifier.testTag("passwordStrength"),
+                value = strength,
+                color = when {
+                    strength < 0.2f -> PasswordStrengthColor.Weak
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfield.md:67]: Double `Weak`, might produce a confusion in a documentation.

File: docs/03-components/05-input/inputfield.md [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+                label = { Text("Strength: $strength") },
+            )
+        },
+    )
+}
+```
+
+## UI testing
+
+Slotting API allows you to add testTag to particular component parts.
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfield.md:81]:
```suggestion
The slotting API allows you to add a testTag to particular component parts.
```

File: docs/03-components/05-input/inputfile.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+---
+title: Android
+---
+
+## Overview
+
+InputFile orbit component is provided only as a visual representation of a clickable field with the almost same API as common TextField.
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfile.md:7]:
```suggestion
`InputFile` Orbit component is provided only as a visual representation of a clickable field with almost the same API as common `TextField`.
```

File: docs/03-components/05-input/inputfile.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+## Usage
+
+Define the text value and click action of the `ClickableField`.
+
+```kotlin
+@Composable
+fun Example() {
+    ClickableField(
+        value = "Attach filed",
+        onClick = { /* ... */ },
```
Comment (Drekorian) [inline docs/03-components/05-input/inputfile.md:19]: _LOGAF_ < 1

 ```suggestion
        onClick = {},
```

File: docs/03-components/05-input/radio.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+title: Android
+---
+
+## Overview
+
+`Radio` component is provided in two flavors, as a simple radio and as a row with an additional label and a
+description:
+
+- [`Radio`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-radio.html) -  simple radio
+- [`RadioField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-radio-field.html) a radio with label and description
```
Comment (Drekorian) [inline docs/03-components/05-input/radio.md:11]:
```suggestion
- [`RadioField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-radio-field.html) a radio with a label and a description
```

File: docs/03-components/05-input/radio.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+## Usage
+
+`Checkbox` accepts a boolean state and a callback on its change.
+
+```kotlin
+@Composable
+fun Example() {
+    Radio(
+        selected = false,
+        onClick = { /*...*/ },
```
Comment (Drekorian) [inline docs/03-components/05-input/radio.md:22]: _LOGAF_ < 1

```suggestion
        onClick = {},
```

File: docs/03-components/05-input/radio.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+`Radio` component is provided in two flavors, as a simple radio and as a row with an additional label and a
+description:
+
+- [`Radio`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-radio.html) -  simple radio
+- [`RadioField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-radio-field.html) a radio with label and description
+
+## Usage
+
+`Checkbox` accepts a boolean state and a callback on its change.
```
Comment (Drekorian) [inline docs/03-components/05-input/radio.md:15]:
```suggestion
`Checkbox` accepts a `Boolean` state and a callback on its change.
```

File: docs/03-components/05-input/radio.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+@Composable
+fun Example() {
+    Radio(
+        selected = false,
+        onClick = { /*...*/ },
+    )
+}
+```
+
+`RadioField` has two additional slots - `description` and `label`.
```
Comment (Drekorian) [inline docs/03-components/05-input/radio.md:27]:
```suggestion
`RadioField` has two additional slots - a `description` and a `label`.
```

File: docs/03-components/05-input/radio.md [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+        ) {
+            Text("StarWars")
+        }
+    }
+}
+```
+
+## UI testing
+
+Slotting API allows you to add testTag to particular component parts. Utilize `assertIsSelected`
```
Comment (Drekorian) [inline docs/03-components/05-input/radio.md:56]:
```suggestion
The Slotting API allows you to add a `testTag` to particular component parts. Utilize `assertIsSelected`
```

File: docs/03-components/05-input/textarea.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
+---
+title: Android
+---
+
+## Overview
+
+Support for a multi-line text input (textarea) is provided
+by [TextField composable](https://orbit.kiwi/components/input/inputfield/android/).
+
+Simply utilize `singleLine`. Optionally you cen set up the maximum of rendered lines by `maxLines` argument.
```
Comment (Drekorian) [inline docs/03-components/05-input/textarea.md:10]:
```suggestion
Simply utilize `singleLine`. Optionally you may set up the maximum of rendered lines by the `maxLines` argument.
```

File: docs/03-components/05-input/select.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+
+## Overview
+
+`SelectField` component is provided as combination of the input and popup list:
+
+- [`SelectField`](https://kiwicom.github.io/orbit-compose/ui/kiwi.orbit.compose.ui.controls/-select.html)
+
+## Usage
+
+`SelectField` API is similar to `TextField`, it accepts the value, `label`, `error` and `info` slots, and more. Pass the `List<T>` options and define their rendering using `optionContent` slot. You may utilize `leadingIcon`, but it cannot be made clickable as the whole field is clickable and opens the popup list.
```
Comment (Drekorian) [inline docs/03-components/05-input/select.md:13]:
```suggestion
`SelectField` API is similar to `TextField`, it accepts the value, `label`, `error` and `info` slots, and more. Pass the `List<T>` options and define their rendering using the `optionContent` slot. You may utilize `leadingIcon`, but it cannot be made clickable as the whole field is clickable and opens the popup list.
```

File: docs/03-components/05-input/select.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+        label = { Text("Pick a letter") }
+    ) { letter ->
+        Text(letter)
+    }
+}
+```
+
+## UI testing
+
+Slotting API allows you to add testTag to particular options. Utilize unmerged tree to click them. The select's value is exposed through EditableText semantic property.
```
Comment (Drekorian) [inline docs/03-components/05-input/select.md:32]:
```suggestion
The slotting API allows you to add `testTag` to particular options. Utilize the unmerged tree to click them. The select's value is exposed through the `EditableText` semantic property.
```

**Review by Drekorian — APPROVED** [review]

**Review by gerak-cz — APPROVED** [review]

**Pull request merged**